_test_unstructured_client/test_split_pdf_hook.py

```diff
@@ -0,0 +1,268 @@
+import io
+import logging
+import os
+from concurrent.futures import Future
+from unittest import TestCase
+
+import requests
+import pytest
+from requests_toolbelt import MultipartDecoder, MultipartEncoder
+
+
+from unstructured_client._hooks.custom import SplitPdfHook
+from unstructured_client.models import shared
+
+
+class TestSplitPdfHook(TestCase):
+
+    def test_unit_sdk_init(self):
+        hook = SplitPdfHook()
+        test_url = "http://localhost:5000"
+        test_client = requests.Session()
+
+        hook.sdk_init(test_url, test_client)
+
+        self.assertEqual(hook.client, test_client)
```

Comment:
Very nice class and all the tests! Just one question, does it require the API to be really running on port 5000 (then it's more like integration than unit test)?

Reply:
Nope, it doesn't. You suggest to change it to some non url like string?

Reply:
Added